lib/main.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'onlyoffice_s3_wrapper'
+require_relative '../data/static_data'
+
+
+# Methods for download files
+class Downloader
+  def initialize
+    FileUtils.makedirs('./tmp')
+    @tmp_dir = './tmp'
+  end
+
+  def s3
+    @s3 ||= OnlyofficeS3Wrapper::AmazonS3Wrapper.new(bucket_name: 'conversion-testing-files', region: 'us-east-1')
+  end
+
+  def download(lib)
+    lib.each do |el|
```

Comment:
`lib` and `el` bad names for variables, 

`lib` is misleading
`el` is too short and can be replaced to another name